basic/escrow/tests/lib.rs

```diff
@@ -0,0 +1,182 @@
+use radix_engine::ledger::*;
+use radix_engine::transaction::*;
+use scrypto::prelude::*;
+
+#[derive(Copy, Clone)]
+struct User {
+    pub_key: Address,
+    address: Address
+}
+
+struct TestUtils<'a, L: Ledger> {
```

Comment:
I like this `TestUtils`

Reply:
Thanks ! I would like to put it into a cargo package but I am not yet good enough with Rust to do it